mesonbuild/compilers/swift.py

```diff
@@ -39,6 +41,10 @@ def __init__(self, exelist: T.List[str], version: str, for_machine: MachineChoic
                          is_cross=is_cross, full_version=full_version,
                          linker=linker)
         self.version = version
+        if self.info.is_darwin():
+            self.sdk_path = subprocess.check_output(['xcrun', '--show-sdk-path'],
+                                                    universal_newlines=True,
+                                                    encoding='utf-8').strip()
```

Comment:
This can fail to run `xcrun` if, for example, the Xcode license has not been accepted. That case needs to be handled gracefully.